src/QMCHamiltonians/ACForce.cpp

```diff
@@ -15,18 +15,23 @@
  */
 #include "ACForce.h"
 #include <sstream>
+#include "OhmmsData/AttributeSet.h"
 
 namespace qmcplusplus
 {
 ACForce::ACForce(ParticleSet& source, ParticleSet& target, TrialWaveFunction& psi_in, QMCHamiltonian& H)
-    : ions(source), elns(target), psi(psi_in), ham(H), FirstForceIndex(-1), Nions(0)
+    : ions(source), elns(target), psi(psi_in), ham(H), FirstForceIndex(-1), Nions(0), useSpaceWarp(false), swt(target,source)
 {
   prefix = "ACForce";
   myName = prefix;
   Nions  = ions.getTotalNum();
```

Comment:
please make Nions const in its definition and assign its value at the constructor initializer list.